tests/net/ieee802154/6lo_fragment/src/main.c

```diff
@@ -438,11 +438,11 @@ static struct net_fragment_data test_data_8 = {
 	.iphc = false
 };
 
-static uint8_t frame_buffer_data[IEEE802154_MTU];
+static uint8_t frame_buffer_data[CONFIG_NET_L2_IEEE802154_MPDU_SIZE];
```

Comment:
Currently this wastes space - FCS is not (yet) reported to upper layers. Should be changed only when introducing this feature in the same change set IMO.